src/appkit.cs

```diff
@@ -18378,11 +18379,17 @@ partial interface NSWindow : NSAnimatablePropertyContainer, NSUserInterfaceItemI
 	
 		[Export ("contentRectForFrameRect:")]
 		CGRect ContentRectFor (CGRect frameRect);
-	
+
+		[Export ("init")]
+		[PostSnippet ("if (!DisableReleasedWhenClosedInConstructor) { ReleasedWhenClosed = false; }")]
```

Comment:
is the condition required ? e.g. `ReleasedWhenClosed = DisableReleasedWhenClosedInConstructor;`